src/containers/custom-procedures.jsx

```diff
@@ -55,11 +56,46 @@ class CustomProcedures extends React.Component {
             const metrics = this.workspace.getMetrics();
             const {x, y} = this.mutationRoot.getRelativeToSurfaceXY();
             const dy = (metrics.viewHeight / 2) - (this.mutationRoot.height / 2) - y;
-            let dx = (metrics.viewWidth / 2) - (this.mutationRoot.width / 2) - x;
-            // If the procedure declaration is wider than the view width,
-            // keep the right-hand side of the procedure in view.
-            if (this.mutationRoot.width > metrics.viewWidth) {
-                dx = metrics.viewWidth - this.mutationRoot.width - x;
+            let dx;
+            if (this.props.isRtl) {
+                // HACK!! This is temporary until we can figure out what's going on width
```

Comment:
We should probably file an issue to come up with a real fix for this and reference it here as a TODO comment e.g. `TODO (#blah)` so that we can keep track of this.